docs/api.md

```diff
@@ -307,6 +322,54 @@ If Google Chrome (rather than Chromium) is preferred, a [Chrome Canary](https://
 >
 > See [`this article`](https://www.howtogeek.com/202825/what%E2%80%99s-the-difference-between-chromium-and-chrome/) for a description
   of the differences between Chromium and Chrome. [`This article`](https://chromium.googlesource.com/chromium/src/+/lkcr/docs/chromium_browser_vs_google_chrome.md) describes some differences for Linux users.
+  
+### class: Downloader
+
+A Downloader can download and manage different versions of Chromium.
```

Comment:
ChromiumDownloader? At first I thought this PR was for handing file downloads, but I see that's it for downloading Chromium. I wonder if the name should be more explicit?

`createDownloader` -> `createChromiumDownloader` too. Thoughts?

Reply:
@ebidel in a follow-up, I'll teach it to download content_shell builds, so it's not about chromium only.